script-docs/hub/ui/release_notes/2025-11-24.rst

```diff
@@ -0,0 +1,103 @@
+2.1.0 (2025-11-24)
+==================
+
+We are releasing a new version of the Hub UI that introduces audit logs, a new task system, enhanced scans, and improved UI and content.
+This helps you manage your evaluation process, track your scans, and improve the collaboration with your team.
+
+Hub UI
+------
+
+What's new?
+~~~~~~~~~~~~
+
+.. raw:: html
+
+    <div style="position: relative; width: 100%; max-width: 100%; padding-top: 56.25%;">
+    <iframe 
+        src="https://player.vimeo.com/video/1140819940?h=7be84b6c71&amp;autoplay=1&amp;muted=1&amp;loop=1&amp;controls=0"
+        frameborder="0"
+        allow="autoplay; fullscreen; picture-in-picture; clipboard-write; encrypted-media; web-share"
+        referrerpolicy="strict-origin-when-cross-origin"
+        title="release-demo"
+        data-ready="true"
+        style="position:absolute;top:0;left:0;width:100%;height:100%;border-radius:10px;box-shadow:0 2px 16px rgba(0,0,0,0.10);"
+        allowfullscreen
+    ></iframe>
+    </div>
+
+**Tasks & Follow-up Workflow**
+   Simplify your team collaboration:
+   
+   * **Create Tasks from Failures** - Create tasks directly from failed evaluations or scans
+   * **Assign Owners with Notifications** - Assign task owners with email notifications (opt-out available)
+   * **Auto-Draft Bad Tests** - When you create a task from a bad test will propose to automatically mark it as Draft
+   * **Hide Noisy Results** - Hide noisy results while still tracking them with tasks
```

Comment:
PRobably better to frame around false positive (hide the result, track the case to fix it)